iOS-13-Migration-Guide.md

```diff
@@ -454,3 +454,7 @@ If your App supports incoming calls, you **MUST** perform the following steps to
 You can reference the 2.1 quickstart for [obj-c](https://github.com/twilio/voice-quickstart-objc/tree/2.x) and [swift](https://github.com/twilio/voice-quickstart-swift/tree/2.x) when migrating your application.
 
 A summary of the API changes and new Insights events can be found in the [2.1.0 changelog](https://www.twilio.com/docs/voice/voip-sdk/ios/2x-changelog#210-september-5-2019).
+
+
+## Register Your App with PushKit at Launch Time
```

Comment:
The wording is fine but I would move this as a step in each migration guide. Finding at the bottom was surprising to me and hard to find.

Reply:
Yeah, @bchen-twilio had the same comment. I have moved the section to top of the migration guide. WDYT @idelgado ?